active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,120 @@
+
+# Background
+
+There’s currently no consistent and Fluent way to address the common UX pattern of a breadcrumb.
+This control is needed for situations where the user's navigation trail (in a file system or menu system) needs
+to be persistently visible.
+A Breadcrumb control is movitated by its use in many app scenarios and supporting developers in migrating from WPF.
+
+This control is similar to an ItemsControl (has an ItemsSource property), but does not derive from it.
+
+## V1/V2 Breadcrumb
+
+The work for Breadcrumb has been scoped to V1 and V2 stages of the API.
+In V1 Breadcrumb no chevrons are interactable to show children.
+V2 Breadcrumb will add functionality to have flyouts from chevrons for every node,
+to view children of that node. This spec focuses on V1 Breadcrumb,
+but keeping in mind that V1 is positioned in a way to have V2 features smoothly added later. 
+
+# API Pages
+
+## Breadcrumb class
+
+Use the Breadcrumb control to private persisting 'trail' that shows the user's navigation path.
```

Comment:
I'm not sure what *private* persisting 'trail' means. Could you clarify?

Reply:
updated this with an "is this the right control", thanks for the catch!